R/merge_hydrofabrics.R

```diff
@@ -182,17 +202,17 @@ assign_global_identifiers <- function(gpkgs                     = NULL,
                                  lu$member_comid)) %>% 
        mutate(member_comid = strsplit(member_comid, split = ",")) %>% 
        unnest('member_comid') %>% 
-       filter(member_comid %in% vpu_topo$COMID) %>% 
        mutate(member_comid = as.integer(member_comid)) %>% 
+       filter(member_comid %in% vpu_topo$COMID) %>% 
```

Comment:
I don't think reordering this is going to break anything but it wouldn't work for me in the other order because the filter was integer comids but the mutate hadn't run yet.